sentry/__manifest__.py

```diff
@@ -3,16 +3,25 @@
 {
     "name": "Sentry",
     "summary": "Report Odoo errors to Sentry",
-    "version": "14.0.1.0.2",
+    "version": "14.0.1.0.0",
     "category": "Extra Tools",
     "website": "https://github.com/OCA/server-tools",
     "author": "Mohammed Barsi,"
     "Versada,"
     "Nicolas JEUDY,"
-    "Odoo Community Association (OCA)",
+    "Odoo Community Association (OCA),"
+    "Vauxoo",
+    "maintainers": ["barsi", "naglis", "versada", "moylop260", "fernandahf"],
```

Comment:
I think there are no rules regarding maintainers quantity, but maybe 5 maintainers is a little too much ? 

CC : @OCA/community-maintainers